pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[build-system]
+requires = ["setuptools >= 67.8.0", 
+            "cython >= 0.29.35",
+            "asn1crypto >= 1.5.1",
```

Comment:
This has mixed build requirements with runtime requirements. Specifically there's two different versions of asn1crypto in play.